python-pard68/main.py

```diff
@@ -64,6 +64,7 @@ def get_highest_scoring(candidates):
 
 	# setup
 	word = sys.argv[1]
+	print(len(word))
```

Comment:
@pard68 this looks like a debugging statement. do you still want this in there?

Reply:
much debugging